README.md

```diff
@@ -134,6 +134,12 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 ## General
 ([Go to top](#awesome4girls---))
 
+ - [ACT-W National](https://act-w.org/)
```

Comment:
You can also add some details about ACT-W-Local chapters.
They have conferences with a similar agenda in multiple cities across the country. The national is the biggest one.